assets/src/edit-story/components/panels/videoAccessibility.js

```diff
@@ -97,22 +96,14 @@ function VideoAccessibilityPanel({ selectedElements, pushUpdate }) {
         <ExpandedTextInput
           placeholder={__('Title', 'web-stories')}
           value={title || ''}
-          onChange={(value) => pushUpdate({ title: value || null })}
+          onChange={(value) =>
+            pushUpdate({ title: value || null, alt: value || null })
```

Comment:
I think a better way would be to just update the title (`pushUpdate({ title: value || null })`) and then in `vide/output.js` simply do `title={title} alt={title}`

Reply:
Ah, good call, that's even better.